### PR TITLE
UG documentation for the segment editor in dataform

### PR DESCRIPTION
**Pull request by SriRadheshNagS**
UG documentation for the segment editor in dataform

Added the segment editor in the UG

### CODE REVIEW

File: MAUI/DataForm/Editors.md [hunk 1]
```diff
@@ -1318,6 +1332,137 @@ public class DataFormItemsSourceProvider : IDataFormSourceProvider
 
 N> [View editors sample in GitHub](https://github.com/SyncfusionExamples/maui-dataform/tree/master/DataFormEditors)
 
+## Segment editor
+
+In the `Segment` editor, the [SfSegmentedControl](https://www.syncfusion.com/maui-controls/maui-segmented-control) will be loaded.
+
+#### Changing the ItemsSource of Segment
+
+The [ItemsSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormListItem.html#Syncfusion_Maui_DataForm_DataFormListItem_ItemsSource) for the radio group editor is generated for the `enum` data type property. In order to add the `Segment` editor in the DataForm, you need to register the editor as `DataFormEditorType.Segment` for the required property by using the [RegisterEditor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_RegisterEditor_System_String_Syncfusion_Maui_DataForm_DataFormEditorType_) method.
```
Comment (saiganesh-sakthivel) [inline MAUI/DataForm/Editors.md:1341]: prepare the content properly, why radio group editor come in the segment control

File: MAUI/DataForm/Editors.md [hunk 2]
```diff
@@ -1318,6 +1332,137 @@ public class DataFormItemsSourceProvider : IDataFormSourceProvider
...
+
+You can display the complex type property values in the picker editor by using the [DisplayMemberPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormListItem.html#Syncfusion_Maui_DataForm_DataFormListItem_DisplayMemberPath) and [SelectedValuePath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormListItem.html?tabs=tabid-1#Syncfusion_Maui_DataForm_DataFormListItem_SelectedValuePath) properties of DataFormSegmentItem. You need to use the [GenerateDataFormItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_GenerateDataFormItem) event to set [DisplayMemberPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormListItem.html#Syncfusion_Maui_DataForm_DataFormListItem_DisplayMemberPath) and [SelectedValuePath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormListItem.html?tabs=tabid-1#Syncfusion_Maui_DataForm_DataFormListItem_SelectedValuePath) property values when [AutoGenerateItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_AutoGenerateItems) is true.
```
Comment (saiganesh-sakthivel) [inline MAUI/DataForm/Editors.md:1407]: change picker into segmentedcontrol

Comment (SriRadheshNagS) [reply MAUI/DataForm/Editors.md:1407]: Changed the content

**Review by saiganesh-sakthivel — APPROVED** [review]
fine

**Review by MuniappanSubramanian — APPROVED** [review]

**Pull request merged**